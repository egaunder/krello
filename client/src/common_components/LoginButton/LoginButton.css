.login__button {
  background-color: #2ecc71;
  border: none;
  outline: none;
  border-radius: 3px;
  line-height: 1.5em;
  font-size: 1em;
  color: white; }
  .login__button:hover {
    background-color: #27ae60;
    transition-duration: 200ms; }
