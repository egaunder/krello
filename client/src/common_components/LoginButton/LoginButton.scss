@import '../../utils/styles/common_colors.scss';

.login__button {
  background-color: $emerald;
  border: none;
  outline: none;
  border-radius: 3px;
  line-height: 1.5em;
  font-size: 1em;
  color: white;
  &:hover {
    background-color: $nephritis;
    transition-duration: 200ms;
  }
}