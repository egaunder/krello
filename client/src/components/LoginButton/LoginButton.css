#login-button {
  background-color: #315989;
  background-color: #298fca;
  color: white; }
  #login-button:hover {
    background-color: #115798; }
