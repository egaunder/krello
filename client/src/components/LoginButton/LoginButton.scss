@import '../../utils/commonCss.scss';

#login-button {
  background-color: $trelloDarkBlue;
  color: white;
}