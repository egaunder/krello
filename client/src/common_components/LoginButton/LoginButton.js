import React from 'react'
import PropTypes from 'prop-types'
import './LoginButton.css'

const LoginButton = props => (
  <button className="login__button" onClick={() => props.onClick()}>Login</button>
)

LoginButton.propTypes = { onClick: PropTypes.func.isRequired }

export default LoginButton
