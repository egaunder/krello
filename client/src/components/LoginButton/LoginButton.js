import React from 'react'
import './LoginButton.css'

const LoginButton = () => (
  <button id="login-button">Login</button>
)

export default LoginButton
