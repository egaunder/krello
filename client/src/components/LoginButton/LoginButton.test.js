import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import LoginButton from './LoginButton'

describe('[LoginButton]', () => {
  test('should render correctly', () => {
    const output = shallow(<LoginButton />)
    expect(shallowToJson(output)).toMatchSnapshot()
  })
})
