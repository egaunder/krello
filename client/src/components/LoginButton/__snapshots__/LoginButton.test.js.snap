// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[LoginButton] should render correctly 1`] = `
<button
  id="login-button"
>
  Login
</button>
`;
